Source/p_spec.c

```diff
@@ -503,12 +503,24 @@ fixed_t P_FindShortestTextureAround(int secnum)
     if (twoSided(secnum, i))
       {
         const side_t *side;
-        if ((side = getSide(secnum,i,0))->bottomtexture > 0 && //jff 8/14/98 texture 0 is a placeholder
-            textureheight[side->bottomtexture] < minsize)
-          minsize = textureheight[side->bottomtexture];
-        if ((side = getSide(secnum,i,1))->bottomtexture > 0 && //jff 8/14/98 texture 0 is a placeholder
-            textureheight[side->bottomtexture] < minsize)
-          minsize = textureheight[side->bottomtexture];
+        if (demo_version < 203)
```

Comment:
Thanks, but isn't this a bit cumbersome? I mean, the only difference here is the `>=` operator. And since we are talking about integer types, the first valid number to fulfill the `>= 0` condition is `0`, whereas the first valid number to fulfill the `> 0` condition is `1`, which in turn is also the first to fulfill `>= 1`. So, it boils down to `0` and `1` switching roles, or am I missing something?
```
int mintex = (demo_version < 203) ? 1 : 0;
```